baremetal/src/Messenger.cc

```diff
@@ -10,103 +10,109 @@
 #include <ebbrt/UniqueIOBuf.h>
 
 uint16_t ebbrt::Messenger::port_;
+double ebbrt::Messenger::Connection::occupancy_ratio_ = 0.2;
+uint8_t ebbrt::Messenger::Connection::preallocate_chain_len_ = 100;
```

Comment:
Are these truly constants? Make these `const constexpr`. We also use the naming convention `kOccupancyRatio` to signify constants.